MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -7926,95 +7926,245 @@ function MAXIS_dialog_navigation()
+    If ButtonPressed = CDEM_button then call navigate_to_MAXIS_screen("CCOL", "CDEM") 'Demand Letter Inquiry (CDEM)
+    If ButtonPressed = CRAA_button then call navigate_to_MAXIS_screen("CCOL", "CRAA") 'Claim Receipts and Adjust (CRAA)
+    If ButtonPressed = CTHW_button then call navigate_to_MAXIS_screen("CCOL", "CTHW") 'Claim Trans Log By Worker (CTHW)
+    If ButtonPressed = CTHC_button then call navigate_to_MAXIS_screen("CCOL", "CTHC") 'Claim Trans Hist By Claim (CTHC)
+    If ButtonPressed = CTHE_button then call navigate_to_MAXIS_screen("CCOL", "CTHE") 'Claim Trans Paid By Case (CTHE)
+    If ButtonPressed = CTHP_button then call navigate_to_MAXIS_screen("CCOL", "CTHP") 'Claim Trans Paid By Person (CTHP)
+    If ButtonPressed = CTOP_button then call navigate_to_MAXIS_screen("CCOL", "CTOP") 'Claim TOP Status (CTOP)
+    If ButtonPressed = CBIL_button then call navigate_to_MAXIS_screen("CCOL", "CBIL") 'Bill Inquiry- By Person (CBIL)
+    If ButtonPressed = PRMX_button then call navigate_to_MAXIS_screen("CCOL", "PRMX") 'Entry of PreMAXIS Claim (PRMX)
+    'REPT
+    If ButtonPressed = USER_button then call navigate_to_MAXIS_screen("REPT", "USER")  'MAXIS User Information (USER)
+    If ButtonPressed = ACTV_button then call navigate_to_MAXIS_screen("REPT", "ACTV")  'Active Caseload (ACTV)
+    If ButtonPressed = PND1_button then call navigate_to_MAXIS_screen("REPT", "PND1")  'GAF I Pending Report (PND1)
+    If ButtonPressed = PND2_button then call navigate_to_MAXIS_screen("REPT", "PND2")  'GAF II Pending Report (PND2)
+    If ButtonPressed = INAC_button then call navigate_to_MAXIS_screen("REPT", "INAC")  'Inactive Caseload (INAC)
+    If ButtonPressed = EOMC_button then call navigate_to_MAXIS_screen("REPT", "EOMC")  'End of Month Closures (EOMC)
+    If ButtonPressed = ARPT_button then call navigate_to_MAXIS_screen("REPT", "ARPT")  'Activity Reports (ARPT)
+    If ButtonPressed = RPTS_button then call navigate_to_MAXIS_screen("REPT", "RPTS")  'County Program Totals (RPTS)
+    If ButtonPressed = MFCM_button then call navigate_to_MAXIS_screen("REPT", "MFCM")  'MFIP Participant Case Management (MFCM)
+    If ButtonPressed = WBCM_button then call navigate_to_MAXIS_screen("REPT", "WBCM")  'Work Benefit Case Mgmt (WBCM)
+    If ButtonPressed = INTR_button then call navigate_to_MAXIS_screen("REPT", "INTR")  'Interstate Match Report (INTR)
+    If ButtonPressed = REVS_button then call navigate_to_MAXIS_screen("REPT", "REVS")  'Review Dates (REVS)
+    If ButtonPressed = REVW_button then call navigate_to_MAXIS_screen("REPT", "REVW")  'Review Report (REVW)
+    If ButtonPressed = FCRV_button then call navigate_to_MAXIS_screen("REPT", "FCRV")  'Foster Care Review Report (FCRV)
+    If ButtonPressed = MRSR_button then call navigate_to_MAXIS_screen("REPT", "MRSR")  'Monthly Reporters Status (MRSR)
+    'If ButtonPressed = GRMR_button then call navigate_to_MAXIS_screen("REPT", "GRMR")  'GRH Status Update (GRMR) ' GRMR IS NO LONGER VALID 05/09/22
```

Comment:
Remove commented out code. You can add the comment to your commit about the panel no longer being valid.